### ISSUE TITLE
Use release section

### ISSUE THREAD

**Issue by rancidfrog** [Kunzisoft/RememBirthday#4]
Have release section with latest release and apk available

**Comment by J-Jamet** [comment]
Yes I just resume the developments, I have not finished, but as soon as a clean version is good, I will deploy.
